msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -176,26 +176,133 @@ TeamsJS is a rich library used to create Tabs using JavaScript. Use the latest T
 
 Microsoft Teams JavaScript library exposes the frameContext in which your meeting tab URL is loaded in the getContext API. The possible values of frameContext are content, task, setting, remove, sidePanel, and meetingStage. This allows you to build customized experiences based on where the app renders. For example, showing a specific collaboration focused UI when in the `meetingStage` and a different meeting preparation UI in the chat tab (`content`). For more information, see [getContext API](/microsoftteams/platform/tabs/how-to/access-teams-context?tabs=teamsjs-v2).
 
+## App caching
+
+App caching helps you to improve subsequent launch time of the apps that are loaded in the meeting side panel.
```

Comment:
Faster App launch makes for a delightful experience and will encourage users to return to your App

Reply:
Updated

Reply:
Please revert to original. This is confusing IMO. I will ask Loki to add wording if needed